controllers/controllers/services/instances/managed/controller_test.go

```diff
@@ -330,6 +342,37 @@ var _ = Describe("CFServiceInstance", func() {
 				}))
 			}).Should(Succeed())
 		})
+
+		It("sets initial state in instance last operation", func() {
+			Eventually(func(g Gomega) {
+				g.Expect(adminClient.Get(ctx, client.ObjectKeyFromObject(instance), instance)).To(Succeed())
+				g.Expect(brokerClient.ProvisionCallCount()).To(BeNumerically(">=", 1))
+				g.Expect(instance.Status.LastOperation).To(Equal(korifiv1alpha1.LastOperation{
+					Type:  "create",
+					State: "initial",
+				}))
+			}).Should(Succeed())
+		})
+	})
+
+	When("polling the initial instance last operation", func() {
```

Comment:
The OSBAPI broker will never return `initial` state, see [spec](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#body-1). This makes the whole `When` redundant.